articles/a-beginners-guide-to-python/index.md

```diff
@@ -0,0 +1,207 @@
+Python is among the most popular programming languages in the world. Python is ranked high in different StackOverflow programming surveys each year. This article helps beginners familiarize themselves with the basic concepts of Python.  
+
+### Introduction
+According to the official [documentation](https://www.python.org/), Python is a high-level and general-purpose programming language. It is a preferred programming language to many developers due to its huge range of libraries. Individuals can use these libraries to create complex applications rather than building everything from scratch. Other reasons why developers love Python is due to its simplicity, flexibility, and versatility. Today, Python is being used in upcoming fields such as machine learning and artificial intelligence.
+
+### What You will learn
+
+In this tutorial, you will learn how to define and use the following elements
+•   Variables
+•   Methods
+•   loops
+•   lists and arrays
+•   Classes
+
+### Prerequisites
+For simplicity, we will be using IDLE in this tutorial. IDLE is the default python code editor and compiler. You can download it from [here](https://www.python.org/downloads/).
+Also, note that this tutorial is based on python 3.9 rather than the 3.6 version.
+
+### Understanding variables
+Variables are simply declarations that are used to store certain values. For instance, the variable `name` can hold the value of `John Smith.` Several rules need to be considered when declaring variable names. For starters, a variable name cannot begin with a number. 
+
+2name  = incorrect #incorrect
+
+name = correct  #correct
+
+Variable names are case sensitive. This means that the variable `school` is not the same as `School`.
+
+Variables can hold different data types. This includes strings, integers, Booleans, long, lists, and arrays.
+
+In Python, we do not need to declare the data type while writing a variable. This is because the code is compiled and interpreted later. The compiler will throw an error in case there is a mismatch in the data types.
+
+Let’s talk about the different data types.
+
+1. Strings
+
+Strings are usually presented in a text format. We declare a string variable, as shown below.
+```
```

Comment:
Please add syntax highlighting to all the code snippets in this article. For example:

```md
```python
print("Hello")
```